examples/json-extension/server/tests/unit/test_features.py

```diff
@@ -17,7 +17,7 @@
 import json
 
 import pytest
-from unittest.mock import Mock
+from mock import Mock
```

Comment:
The line I removed here works locally for me, but appveyor hates it and needs the added line..?

Reply:
That import is needed because `Mock` is used on several places in that module (see [test_features.py line 41](https://github.com/openlawlibrary/pygls/blob/master/examples/json-extension/server/tests/unit/test_features.py#L41)).